rest/model/project.go

```diff
@@ -68,7 +54,7 @@ func (t *APITriggerDefinition) BuildFromService(h interface{}) error {
 	switch h.(type) {
 	case model.TriggerDefinition:
 		triggerDef = h.(model.TriggerDefinition)
-	case *model.PeriodicBuildDefinition:
```

Comment:
why do we no longer need case *model.PeriodicBuildDefinition:?

Reply:
This was a copypaste error. We wouldn't be converting a periodic build type to a trigger type.